src/Protocol/Packetizer.cpp

```diff
@@ -6,6 +6,7 @@
 #include "Globals.h"
 #include "Packetizer.h"
 #include "../UUID.h"
+#include "../CompositeChat.h"
```

Comment:
Please use imports relative to `src/`

we agreed to change that some time ago but didn't want create a giant commit just to do this